### ISSUE TITLE
Readme link spackpm.herokuapp.com leads to broken page

### ISSUE THREAD

**Issue by ian-bertolacci** [LLNL/spack#17513]
the link: https://spackpm.herokuapp.com/ in the readme goes to a "Application error" page.

> Application error
>An error occurred in the application and your page could not be served. If you are the application owner, check your logs for details. You can do this from the Heroku CLI with the command heroku logs --tail

**Comment by alalazo** [comment]
Added the unreproducible tag, since what I get right now is:
![Screenshot from 2020-07-15 09-02-31](https://user-images.githubusercontent.com/4199709/87514055-f317ff00-c679-11ea-9530-b06b5826d187.png)

Feel free to remove the tag (or ping me to do that) if the error persists for you.

**Comment by tgamblin** [comment]
Something (not sure what) went wrong with the old app and its legacy Slack token.  We started getting bad logins.

So I tried fixing by deploying this app: https://github.com/outsideris/slack-invite-automation, which apparently @alalazo immediately saw 😄.  Unfortunately, the badge seemed like it was displaying the wrong number of users, and it always said 0 were online.

I was able to just redeploy `slackin` (https://github.com/rauchg/slackin), which is what we were using before, and we seem to be back in business.  And the badge displays the correct number of users.

<img width="448" alt="Screen Shot 2020-07-15 at 12 05 47 AM" src="https://user-images.githubusercontent.com/299842/87514642-69dbda80-c62f-11ea-8620-c174030026c7.png">

Comment here if there are further issues.
